Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -194,6 +195,17 @@ public class PBXProjGenerator {
             )
             derivedGroups.append(group)
         }
+        
+        if !bundleFiles.isEmpty {
```

Comment:
This creates a derived group for bundles that are dependencies, similar to the `Frameworks` and `Products` derived groups. We need to do this because otherwise the file reference temporary IDs (`TEMP_blahblahblah`) will not be deterministically generated by the `XcodeProj.ReferenceGenerator`, thus resulting in a new reference value each time `XcodeGen` is run. This is how `XcodeProj` generates these IDs for implicitly linked frameworks as well.